repos/base-linux/src/core/platform_services.cc

```diff
@@ -0,0 +1,39 @@
+/*
+ * \brief  Platform specific services for Linux
+ * \author Johannes Kliemann
+ * \date   2017-11-08
+ */
+
+/*
+ * Copyright (C) 2012-2017 Genode Labs GmbH
+ *
+ * This file is part of the Genode OS framework, which is distributed
+ * under the terms of the GNU Affero General Public License version 3.
+ */
+
+/* Genode includes */
+#include <base/service.h>
+
+/* core includes */
+#include <core_env.h>
+#include <platform.h>
+#include <platform_services.h>
+#include <io_port_root.h>
+
+#include <core_linux_syscalls.h>
+
+/*
+ * Add x86 specific ioport service
+ */
+void Genode::platform_add_local_services(Rpc_entrypoint*,
+                                         Sliced_heap *sliced_heap,
+                                         Registry<Service> *local_services)
+{
+        if(!lx_iopl(3)){
+        	static Io_port_root io_port_root(core_env()->pd_session(),
```

Comment:
whitespace: use tabs for indentation, spaces for alignment. See https://genode.org/documentation/developer-resources/coding_style#Indentation

There are more occurrences of such whitespace issues. It would be nice of you to recheck.